src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -458,6 +458,14 @@ internal string GetClusterTransactionId()
             Debug.Assert(Sharding.Shards.All(s => s.ClusterTransactionIdBase64.Equals(Sharding.Shards[0].ClusterTransactionIdBase64)));
             return Sharding.Shards[0].ClusterTransactionIdBase64;
         }
+
+        internal static string GetKeyForDeletionInProgress(string node, int? shard)
```

Comment:
by convention we agreed to use `shardNumber` as a name instead of `shard` :)

Reply:
Fixed :)